committee-steering/governance/annual-reports.md

```diff
@@ -62,45 +62,135 @@ of being more explicit than using our upstream shorthand or abbreviations.
 a pull request at the end. 
 
-## Questions for report: {#questions}
-
-#### Special Interest Groups:
-##### Operational 
-- How are you doing with operational tasks in [SIG]-governance.md?
-  - Is your README accurate? have a CONTRIBUTING.md file?
-  - All subprojects correctly mapped and listed in sigs.yaml?
-  - What’s your meeting culture? Large/small, active/quiet, learnings? Meeting 
-  notes up to date? Are you keeping recordings up to date/trends in community 
-  members watching recordings?
-- How does the group get updates, reports, or feedback from subprojects? Are 
-there any springing up or being retired? Are OWNERS.md files up to date in these
- areas?
-- Same question as above but for working groups.
-- When was your last public community-wide update? (provide link to deck and/or
- recording)
-
-##### Membership
-- Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
-- How do you measure membership? By mailing list members, OWNERs, or something
-else?
-- How does the group measure reviewer and approver bandwidth? Do you need help
-in any area now? What are you doing about it?
-- Is there a healthy onboarding and growth path for contributors in your SIG?
-What are some activities that the group does to encourage this? What programs
-are you participating in to grow contributors throughout the contributor ladder?
-- What programs do you participate in for new contributors?
-- Does the group have contributors from multiple companies/affiliations? Can end
- users/companies contribute in some way that they currently are not?
-
-##### Current initiatives and project health
-- What are initiatives that should be highlighted, lauded, shout outs, that
-your group is proud of? Currently underway? What are some of the longer tail
-projects that your group is working on?
-- Year to date KEP work: What's now stable? Beta? Alpha? Road to alpha?
-- What initiatives are you working on that aren't being tracked in KEPs?
-- What areas and/or subprojects does the group need the most help with?
-- What metrics/community health stats does your group care about and/or measure?
-Examples?     
+## Questions for report:
+
+### Special Interest Groups:
+
+```
+# {sig-name} - {YYYY} annual report
+
+## Current initiatives
+
+1. What work did the SIG do this year that should be highlighted?
+
+   - 
+   - 
+   - 
+
+2. What initiatives are you working on that aren't being tracked in KEPs?
+
+   - 
+   - 
+   - 
+
+3. KEP work in {YYYY} (1.x, 1.y, 1.z): (generated)
+   - Stable
+      - [{name}]({KEP link}) - {release}
+      - [{name}]({KEP link}) - {release}
+   - Beta
+      - [{name}]({KEP link}) - {release}
+      - [{name}]({KEP link}) - {release}
+   - Alpha
+      - [{name}]({KEP link}) - {release}
+      - [{name}]({KEP link}) - {release}
+   - Pre-alpha
+      - [{name}]({KEP link})
+
+## Project health
+
+1. What areas and/or subprojects does your group need the most help with? (link to more details)
```

Comment:
```suggestion
1. What areas and/or subprojects does your group need the most help with? Any areas with 2 or less OWNERs? (link to more details)
```

Reply:
I'm ok putting that here for now, but OWNER file analysis seems like something a tool could do eventually. I'm more interested in a qualitative answer of "we have OWNERS file coverage, but maintainers of X and Y subprojects are stretched thin"